src/main/java/com/uber/cadence/serviceclient/WorkflowServiceTChannel.java

```diff
@@ -207,6 +215,45 @@ private static String getEndpoint(String service, String method) {
     return buildThriftRequest(apiName, body, null);
   }
 
+  @Override
+  public CompletableFuture<Boolean> isHealthy() {
+    final ThriftRequest<Meta.health_args> req =
+        new ThriftRequest.Builder<Meta.health_args>("cadence-frontend", "Meta::health")
```

Comment:
I think hardcoding `"cadence-frontend"` will not work for everyone. `cadence-frontend` just happen to be the default service name for frontend service. For example, in Uber because there are multiple cadence clusters, they have to name it `cadence-frontend-clusterA` `cadence-frontend-clusterB`.  Is this just required for `Meta::health`. How does other TChannel method works?